tests/ca.key

```diff
@@ -0,0 +1,6 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
could you provide a `tests/HACKING.md` or similar document that describes how to generate all these certificates?